### DISCUSSION TITLE
What happens if values don’t change?

### DISCUSSION THREAD

**Discussion by joba-1** [pypolestar/polestar_api#141] [Q&A]
I use your great integration to mainly monitor the battery charge percentage. 
The HA Influx integration saves the values in an Influx table where a grafana dashboard displays the graph and the current value.

The problem is that if there is no new value in the selected time range of the grafana panel then there is no data displayed at all.

What would help would be a new value in Influx every minute, even if the value does not change. Could this integration somehow do this or do I need to tweak the Influx integration of HA?

**Comment by leeyuentuen** [comment]
I'm not sure, but the data on HA will only be changed if the value changes. so if the car is parked and not using battery power, it will keep the previous value. It's only changing if it is charging while it is in driving mode

**Reply by dgomes** [reply]
That used to be true, but since https://developers.home-assistant.io/blog/2024/03/20/state_reported_timestamp you can update more often

**Reply by joba-1** [reply]
not sure, but this looks like it requires changes in the integration.

**Comment by joba-1** [comment]
ok, thanks. Then I’ll have to explore what can be done to distinguish unchanged values from no new values due to an error somewhere in the pipeline.
